src/frontend/js/widgets/SyllabusCourseRunsList/SyllabusAsideList.tsx

```diff
@@ -0,0 +1,158 @@
+import { SyllabusCourseRun } from 'widgets/SyllabusCourseRunsList/SyllabusCourseRun';
+
+const messages = defineMessages({
+  otherCourseRuns: {
+    id: 'components.SyllabusAsideList.otherCourseRuns',
+    description:
+      'Message displayed on the top of course runs list on syllabus when there is only one course run opened',
+    defaultMessage: 'Other course runs',
+  },
+  courseRunsTitle: {
+    id: 'components.SyllabusAsideList.courseRunsTitle',
+    description:
+      'Message displayed on the top of course runs list on syllabus when there is 0 or multiple course runs opened',
+    defaultMessage: 'Course runs',
+  },
+  noCourseRuns: {
+    id: 'components.SyllabusAsideList.noCourseRuns',
+    description: 'Message displayed on on syllabus when there are no course runs to show',
+    defaultMessage: 'No course runs',
+  },
+  noOtherCourseRuns: {
+    id: 'components.SyllabusAsideList.noOtherCourseRuns',
+    description:
+      'Message displayed on on syllabus when there are no other course runs to show than the only one opened',
+    defaultMessage: 'No course runs',
+  },
+  toBeScheduled: {
+    id: 'components.SyllabusAsideList.toBeScheduled',
+    description: 'Message displayed on on syllabus when there are course runs to be scheduled',
+    defaultMessage: 'To be scheduled',
+  },
+  upcoming: {
+    id: 'components.SyllabusAsideList.upcoming',
+    description: 'Message displayed on on syllabus when there are upcoming course runs',
+    defaultMessage: 'Upcoming',
+  },
+  ongoing: {
+    id: 'components.SyllabusAsideList.ongoing',
+    description: 'Message displayed on on syllabus when there are ongoing course runs',
+    defaultMessage: 'Ongoing',
+  },
+  archived: {
+    id: 'components.SyllabusAsideList.archived',
+    description: 'Message displayed on on syllabus when there are archived course runs',
+    defaultMessage: 'Archived',
+  },
+});
+
+export const SyllabusAsideList = ({
+  courseRuns,
+  course,
+}: {
+  courseRuns: CourseRun[];
+  course: Course;
+}) => {
+  const openedRuns = courseRuns.filter((run) =>
+    [Priority.ONGOING_OPEN, Priority.FUTURE_OPEN, Priority.ARCHIVED_OPEN].includes(
+      run.state.priority,
+    ),
+  );
+
+  const otherRuns = courseRuns.filter((run) =>
+    [
+      Priority.TO_BE_SCHEDULED,
+      Priority.FUTURE_NOT_YET_OPEN,
+      Priority.FUTURE_CLOSED,
+      Priority.ARCHIVED_CLOSED,
+    ].includes(run.state.priority),
+  );
+
+  const toBeScheduledRuns = courseRuns.filter((run) =>
+    [Priority.TO_BE_SCHEDULED].includes(run.state.priority),
+  );
+
+  const upcomingRuns = courseRuns.filter((run) =>
+    [Priority.FUTURE_NOT_YET_OPEN].includes(run.state.priority),
+  );
+
+  const ongoingRuns = courseRuns.filter((run) =>
+    [Priority.FUTURE_CLOSED, Priority.ONGOING_CLOSED].includes(run.state.priority),
+  );
+
+  const archivedRuns = courseRuns.filter((run) =>
+    [Priority.ARCHIVED_CLOSED].includes(run.state.priority),
+  );
```

Comment:
Just for readability, what do you think to filter accross `otherRuns` instead of `courseRuns` ?